include/circt/Dialect/FIRRTL/FIRRTLDeclarations.td

```diff
@@ -94,7 +94,7 @@ def NodeOp : FIRRTLOp<"node", [NoSideEffect, SameOperandsAndResultType]> {
   let arguments = (ins PassiveType:$input, OptionalAttr<StrAttr>:$name);
   let results = (outs FIRRTLType:$result);
 
-  let assemblyFormat = "$input attr-dict `:` type($input)";
+  let assemblyFormat = "$input custom<Fancy>(attr-dict) `:` type($input)";
```

Comment:
Please name this soemthing more specific than "Fancy". Maybe `custom<ImplicitSSAName>` ?

Reply:
Thanks. I changed it.